packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -1055,15 +1061,6 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         emit SpamDeletionProposalBondChanged(_spamDeletionProposalBond);
     }
 
-    /**
-     * @notice Get the spam deletion request by the proposal id.
-     * @param spamDeletionRequestId spam deletion request id.
-     * @return SpamDeletionRequest the spam deletion request.
-     */
-    function getSpamDeletionRequest(uint256 spamDeletionRequestId) external view returns (SpamDeletionRequest memory) {
```

Comment:
This function wasn't used anywhere and it could be replaced by making `spamDeletionProposals` public
Doing this gives us extra space under the EIP170 limit